### PR TITLE
Rebrand and PHP compatibility

### PR DESCRIPTION
**Pull request by semsudin**
Tested on OpenMage LTS 19.4.8 and PHP 7.3.27

### CODE REVIEW

File: app/code/community/Qenta/CheckoutPage/Block/Admin/Support/Edit/Form.php [hunk 1]
```diff
@@ -40,28 +40,28 @@ protected function _prepareForm()
...
-            'label'    => Mage::helper('wirecard_checkoutpage')->__('To'),
+            'label'    => Mage::helper('qenta_checkoutpage')->__('To'),
             'class'    => 'required-entry',
             'required' => true,
             'name'     => 'to',
             'options'  => array(
-                'support.at@wirecard.com' => 'Support AT',
-                'support@wirecard.com' => 'Support DE'
+                'support@qenta.com' => 'Support AT',
+                'support@qenta.com' => 'Support DE'
```
Comment (jakubpolomsky) [inline app/code/community/Qenta/CheckoutPage/Block/Admin/Support/Edit/Form.php]: We don't need a select dropdown for the mail address as there is no real selection. Please turn it into a readonly input

File: app/code/community/Qenta/CheckoutPage/Model/System/Config/InvoiceProviders.php [hunk 2]
```diff
@@ -30,14 +30,14 @@
...
-class Wirecard_CheckoutPage_Model_System_Config_InvoiceProviders
+class Qenta_CheckoutPage_Model_System_Config_InvoiceProviders
 {
     public function toOptionArray()
     {
         $themes = array(
             array('value' => 'payolution', 'label' => 'payolution'),
             array('value' => 'ratepay', 'label' => 'RatePay'),
-            array('value' => 'ratepay', 'label' => 'Wirecard'),
+            array('value' => 'ratepay', 'label' => 'Qenta'),
```
Comment (jakubpolomsky) [inline app/code/community/Qenta/CheckoutPage/Model/System/Config/InvoiceProviders.php]: This label cannot be changed to Qenta as Qenta is not an invoice provider. Please rather remove this line

Comment (semsudin) [reply app/code/community/Qenta/CheckoutPage/Model/System/Config/InvoiceProviders.php]: Resolved

File: app/code/community/Qenta/CheckoutPage/Block/Admin/Support/Edit/Form.php (hunk 1 above)
Comment (semsudin) [reply app/code/community/Qenta/CheckoutPage/Block/Admin/Support/Edit/Form.php]: Resolved

**Pull request merged**